src/clients/MultiCallerClient.ts

```diff
@@ -513,3 +514,138 @@ export class MultiCallerClient {
     }
   }
 }
+
+export class TryMulticallClient extends MultiCallerClient {
+  override _buildMultiCallBundle(transactions: AugmentedTransaction[]): AugmentedTransaction {
+    const txn = super._buildMultiCallBundle(transactions);
+    txn.method = "tryMulticall";
+    return txn;
+  }
+
+  protected override async _executeTxnQueue(
+    chainId: number,
+    txns: AugmentedTransaction[] = [],
+    _valueTxns: AugmentedTransaction[] = [],
+    simulate = false
+  ): Promise<TransactionResponse[]> {
+    // Lint Appeasement
+    _valueTxns;
+    const nTxns = txns.length;
+    if (nTxns === 0) {
+      return [];
+    }
+
+    const networkName = getNetworkName(chainId);
+    this.logger.debug({
+      at: "TryMulticallClient#executeTxnQueue",
+      message: `${simulate ? "Simulating" : "Executing"} ${nTxns} transaction(s) on ${networkName}.`,
+    });
+
+    const buildRawTransaction = (contract: Contract, data: string) => {
+      return {
+        contract,
+        data,
+      } as RawTransaction;
+    };
```

Comment:
idk whether this closure is even needed, but it should be possible to squash it pretty severely whilst still retaining type assurances.

```suggestion
    const buildRawTransaction = (contract: Contract, data: string): RawTransaction => {{ contract, data });
```